test/e2e/lib/driver-helper.js

```diff
@@ -530,7 +529,7 @@ export async function selectElementByText( driver, selector, text ) {
 		const allElements = await driver.findElements( selector );
 		return await webdriver.promise.filter( allElements, getInnerTextMatcherFunction( text ) );
 	};
-	return await this.clickWhenClickable( driver, element, null, `while looking for '${ text }'` );
```

Comment:
I'm removing this only temporarily. Already working on a refactor that would include displaying similar message in https://github.com/Automattic/wp-calypso/pull/50944.